nomad/eval_endpoint_test.go

```diff
@@ -851,6 +851,218 @@ func TestEvalEndpoint_List_Blocking(t *testing.T) {
 	}
 }
 
+func TestEvalEndpoint_List_PaginationFiltering(t *testing.T) {
+	t.Parallel()
+	s1, _, cleanupS1 := TestACLServer(t, nil)
+	defer cleanupS1()
+	codec := rpcClient(t, s1)
+	testutil.WaitForLeader(t, s1.RPC)
+
+	// create a set of evals with a known lexigraphical sorting order
+	// and field values to filter on
```

Comment:
In addition to spelling lexicographical wrong, this isn't lexicographical order, it's byte-ordered (as it should be). 😁 Need to fix.